matrix/common/dss_zarr_store.py

```diff
@@ -137,6 +140,14 @@ def _get_dss_client(self, dss_instance="integration"):
         client = hca.dss.DSSClient(config=dss_config)
         return client
 
+    def _validate_zarr(self):
+        expected_fields = ['expression', 'cell_id', 'cell_metadata_string', 'cell_metadata_numeric',
+                           'cell_metadata_string_name', 'cell_metadata_numeric_name', 'gene_id', 'gene_metadata',
+                           'gene_metadata_name']
```

Comment:
It seems like this is a repeat of the properties of this class. It would better if that list existed in only one place...though I'm not sure of the best way to do that.

Reply:
Yea, this was one of the use cases for the enum I wanted to introduce with the zarr store/group refactor. I was thinking we'd clean this up when we introduce that refactor